gate-oauth2/src/main/java/com/netflix/spinnaker/gate/security/oauth2/SpinnakerOIDCUser.java

```diff
@@ -69,7 +101,7 @@ public String getName() {
 
   @Override
   public Map<String, Object> getClaims() {
-    return this.attribute;
+    return this.attributes;
```

Comment:
This could use some comments.  Why is it correct to return attributes in a method called `getClaims` ?

Reply:
Added javadoc
https://github.com/spinnaker/gate/pull/1887/commits/2b67a513a18c7121e81510a3ed10573acb5603db